### DISCUSSION TITLE
Error: No root file handle

### DISCUSSION THREAD

**Discussion by connorferster** [jupyterlite/jupyterlite#1431] [Q&A] [answered]
I have just launched a Jupyterlite site using https using the xeus kernel. It seems to work well. 

However, when I click the Xeus icon in the Launcher to create a new notebook, I am greeted with an error window:

```
Error

No root file handle
```

I have not encountered this before when building my site locally and serving it up with `http.server`. Has anyone seen this before? Any idea what it means?

**Comment by connorferster** [comment]
Interestingly, if I go to the notebook interface by accessing /tree then I can create a new notebook without a problem! It seems to only be a problem in the lab interface.

I also have the file system access plugin installed. I _can_ create a new notebook from the Lab interface in the file system access. It seems that this error only happens when I want to create a notebook in the browser storage (i.e. in the most intuitive way).

For reference (temporarily included), the site is accessible here: https://launch.structuralpython.com

**Comment by jtpio** [comment, marked as answer]
@connorferster do you have this extension installed in your JupyterLite site? https://github.com/jupyterlab-contrib/jupyterlab-filesystem-access

If so this is likely the same issue as https://github.com/jupyterlab/jupyterlab/issues/15629, related to the drive keeping the focus.

**Reply by connorferster** [reply]
This is exactly what is happening. When I click inside the file browser first and _then_ click a kernel tile in the Launcher, there is no problem. 

Thank you @jtpio!
